src/pages/index.page.tsx

```diff
@@ -70,19 +73,7 @@ const Home = () => {
             </div>
           </div>
 
-          <form
```

Comment:
Why are you getting rid of the form submit? 
it is better UX to have a submit event instead of just a button.

Reply:
this was by mistake. I think it was due to a cmd + z spam